blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServer.scala

```diff
@@ -158,14 +162,14 @@ class BlazeBuilder(
 
           var lb = LeafBuilder(
             if (isHttp2Enabled) http2Stage(engine)
-            else http1Stage
+            else http1Stage(true)
```

Comment:
I believe that such argument should use a named-parameter syntax: `http1Stage(secure = true)`